### PR TITLE
Updated otel to v0.63.0

### PR DESCRIPTION
**Pull request by cpheps**
Updated to [Otel v0.63.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.63.0)

### CODE REVIEW

File: integrationtest/hostmetrics_test.go [hunk 1]
```diff
@@ -167,7 +167,8 @@ func loadObservedMetrics(t *testing.T, metricsJSONPath string) pmetric.Metrics {
 
 	t.Logf("Found %v bytes of data at %s, selecting %v bytes", len(data), metricsJSONPath, len(secondBatch))
 
-	metrics, err := pmetric.NewJSONUnmarshaler().UnmarshalMetrics(secondBatch)
+	unmashaller := &pmetric.JSONUnmarshaler{}
```
Comment (braydonk) [inline integrationtest/hostmetrics_test.go:170]: nit: Looks to be a type, `unmarshaller` instead?

**Review by braydonk — APPROVED** [review]
LGTM, if you don't mind fixing the typo that would be good though

File: integrationtest/hostmetrics_test.go (hunk 1 above)
Comment (cpheps) [reply integrationtest/hostmetrics_test.go:170]: Yup, typed to fast will fix.

Comment (cpheps) [reply integrationtest/hostmetrics_test.go]: @braydonk Resolved.

**Review by braydonk — APPROVED** [review]

**Pull request merged**